attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModelTest.java

```diff
@@ -255,10 +258,12 @@ public void testAttributeDefinitionBackwardsCompatibility() throws IOException {
             + "\"id\":\"EVENT.key\","
             + "\"value_kind\":\"TYPE_STRING\","
             + "\"display_name\":\"Display\","
+            + "\"description\":\"\","
```

Comment:
If it's without definitionAndDescription, we don't need an empty `""` string, right? Are we missing something?

Can you add an explicit test `testDescriptionBackwardsCompatibility`?

Reply:
>If it's without definitionAndDescription, we don't need an empty "" string, right? Are we missing something?

I think this is because the default string value is an empty string in protobuf, and this test has two parts - one from `fromJson`, and the second from `fromDTO`. Got it.

Reply:
Can you add an explicit test testDescriptionBackwardsCompatibility?

Reply:
I've reverted the `testAttributeDefinitionBackwardsCompatibility` to it's original state and added new tests in `testAttributeDescriptionBackwardsCompatibility`